metacat-connector-hive/build.gradle

```diff
@@ -13,6 +13,7 @@
 
 dependencies {
     compile project(':metacat-common-server')
+    testCompile project(':metacat-common-server').sourceSets.test.output
```

Comment:
To allow the test to use the static testing data provided in other module. If we have any other way to achieve this, I will make the change to use it.